include/ginkgo/core/base/executor.hpp

```diff
@@ -1027,6 +1095,17 @@ class CudaExecutor : public detail::ExecutorBase<CudaExecutor>,
 
     GKO_ENABLE_FOR_ALL_EXECUTORS(GKO_OVERRIDE_RAW_COPY_TO);
 
+    GKO_DEFAULT_OVERRIDE_VERIFY_MEMORY_TO(OmpExecutor);
+
+    GKO_DEFAULT_OVERRIDE_VERIFY_MEMORY_TO(HipExecutor);
```

Comment:
`#if defined(GINKGO_HIP_PLATFORM_NVCC)` and both have the same id I would also return true.